config/init.sql

```diff
@@ -0,0 +1,14 @@
+CREATE DATABASE prover_db;
```

Comment:
Can you rename the file `init.sql` ==> `initproverdb.sql`?

Reply:
There is only one SQL file that can be attached to initialize the postgres container with whatever we need, and it's common practice to name it **init.sql**. In example, if tomorrow we want to create additional users for each component (sequencer, agregator, etc), that would be done in this file as well, so I would prefer to avoid tying the filename to the specific action it is doing right now, since in fact it could change at any moment. Are you ok with this?